homeassistant/components/sensor/rest.py

```diff
@@ -174,11 +178,13 @@ def device_state_attributes(self):
 class RestData:
     """Class for handling the data retrieval."""
 
-    def __init__(self, method, resource, auth, headers, data, verify_ssl):
+    def __init__(self, method, resource, auth, headers, data, verify_ssl,
+                 timeout=DEFAULT_TIMEOUT):
```

Comment:
Why do we need the default value here?

Reply:
`RestData` class is used in other places, like [pvoutput](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/pvoutput.py#L57). I didn't want to touch other sensors with this change

Reply:
Oh. That's a good point. Such imports are actually not allowed, for this very reason :-/